packages/sdk/contractkit/src/wrappers/FederatedAttestations.ts

```diff
@@ -1,8 +1,170 @@
+import { CeloTransactionObject } from '@celo/connect'
 import { FederatedAttestations } from '../generated/FederatedAttestations'
-import { BaseWrapper } from './BaseWrapper'
+import { BaseWrapper, proxyCall, proxySend } from './BaseWrapper'
 
-// TODO ASv2 -- add params or delete config if there are none
-// & delete other commented-out usages of this Config in CK
-// export interface FederatedAttestationsConfig {}
+type Address = string
+type Identifier = string | number[]
 
-export class FederatedAttestationsWrapper extends BaseWrapper<FederatedAttestations> {}
+export class FederatedAttestationsWrapper extends BaseWrapper<FederatedAttestations> {
+  /**
+   * @notice Returns identifiers mapped to `account` by signers of `trustedIssuers`
+   * @param account Address of the account
+   * @param trustedIssuers Array of n issuers whose identifier mappings will be used
+   * @return countsPerIssuer Array of number of identifiers returned per issuer
+   * @return identifiers Array (length == sum([0])) of identifiers
+   * @dev Adds identifier info to the arrays in order of provided trustedIssuers
+   * @dev Expectation that only one attestation exists per (identifier, issuer, account)
+   */
+  lookupIdentifiers: (
+    account: Address,
+    trustedIssuers: Address[]
+  ) => Promise<{
+    countsPerIssuer: string[]
+    identifiers: string[]
+    '0': string[]
+    '1': string[]
+  }> = proxyCall(this.contract.methods.lookupIdentifiers)
+
+  /**
+   * @notice Helper function for lookupAttestations to calculate the total number of
+   * attestations completed for an identifier by each trusted issuer
+   * @param identifier Hash of the identifier
+   * @param trustedIssuers Array of n issuers whose attestations will be included
+   * @return totalAttestations Sum total of attestations found
+   * @return countsPerIssuer Array of number of attestations found per issuer
+   */
+  lookupAttestations: (
+    identifier: Identifier,
+    trustedIssuers: Address[]
+  ) => Promise<{
+    countsPerIssuer: string[]
+    accounts: string[]
+    signers: string[]
+    issuedOns: string[]
+    publishedOns: string[]
+    '0': string[]
```

Comment:
what are these number fields for?

Reply:
These were the generate parameters for the solidity file - they have been removed